### PR TITLE
[skip ci] Update links and fix dead links.

### PR DESCRIPTION
**Pull request by ireneontheway5**
Signed-off-by: ireneontheway5 <qingying.hu@zilliz.com>

**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [x] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

Fixes #

**What this PR does / why we need it:**

Not Available

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

**Review by JinHai-CN — DISMISSED** [review]

File: SUPPORT.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 We use GitHub for tracking bugs and feature requests. If you need any support for using Milvus, please refer to the following resources below.
 
 ## Documentation
--   [User Documentation](https://www.milvus.io/docs/guides/get_started/install_milvus/install_milvus.md)
--   [Troubleshooting Guide](https://www.milvus.io/docs/v0.6.0/guides/troubleshoot.md)
+-   [User Documentation](https://www.milvus.io/docs/install_milvus.md)
+-   [Troubleshooting Guide](https://www.milvus.io/docs/troubleshoot.md)
 -   [FAQ](https://www.milvus.io/docs/v0.6.0/faq/operational_faq.md)
```
Comment (ireneontheway5) [inline SUPPORT.md:8]:
```suggestion
-   [FAQ](https://www.milvus.io/docs/operational_faq.md)
```

File: docs/test_report/milvus_ivfsq8h_test_report_detailed_version_cn.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ Milvus设置：
 - gpu_search_threshold: 1200
 - search_resources: cpu, gpu0, gpu1
 
-Milvus设置的详细定义可以参考 https://milvus.io/docs/en/reference/milvus_config/ 。
+Milvus设置的详细定义可以参考 https://milvus.io/docs/milvus_config.md 。
```
Comment (ireneontheway5) [inline docs/test_report/milvus_ivfsq8h_test_report_detailed_version_cn.md:99]:
```suggestion
Milvus设置的详细定义可以参考 https://milvus.io/cn/docs/milvus_config.md 。
```

File: docs/test_report/ivfflat_test_report_cn.md [hunk 3]
```diff
@@ -97,7 +97,7 @@ Milvus设置：
 - gpu_cache_capacity: 6
 - use_blas_threshold: 2100
 
-Milvus设置的详细定义可以参考 https://milvus.io/docs/en/reference/milvus_config/ 。
+Milvus设置的详细定义可以参考 https://milvus.io/cn/docs/milvus_config.md 。
```
Comment (PahudPlus) [inline docs/test_report/ivfflat_test_report_cn.md:100]: Remove additional space between md and the full stop.

**Review by PahudPlus — CHANGES_REQUESTED** [review]

**Review by wxyucs — APPROVED** [review]

**Comment by JinHai-CN** [comment]
LGTM!

**Review by JinHai-CN — APPROVED** [review]

**Review by XuPeng-SH — APPROVED** [review]

**Review by yhmo — APPROVED** [review]

**Pull request merged**